pixleesdk/src/main/java/com/pixlee/pixleesdk/client/PXLClient.kt

```diff
@@ -14,7 +14,7 @@ import okhttp3.MediaType.Companion.toMediaTypeOrNull
 /**
  * Created by sungjun on 9/18/20.
  */
-class PXLClient(val context: Context) {
```

Comment:
This should be removed. I decompiled this Kotlin code to java bytecode. this variable in the constructor becomes final variable which leads to a memory leak. 

```java
public final class PXLClient {
   @NotNull
   private final Lazy ktxBasicDataSource$delegate;
   @NotNull
   private final Lazy ktxAnalyticsDataSource$delegate;
   @NotNull
   private final Lazy basicDataSource$delegate;
   @NotNull
   private final Lazy analyticsDataSource$delegate;
   @NotNull
   private final Context context;
```